server/views/pages/error.njk

```diff
@@ -1,12 +1,31 @@
 {% extends "../partials/layout.njk" %}
 
-{% set pageTitle = applicationName + " - Error" %}
+{% block pageTitle %}
+  {{ locale.page.title }}
+{% endblock %}
+
 {% set mainClasses = "app-container govuk-body" %}
 
-{% block content %}
+{% block header %}
+  {% include "../partials//header.njk" %}
```

Comment:
double slash?

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/227/commits/c833e7674387179e5b647d82cdbd785fd0a901ce